store/index.js

```diff
@@ -10,25 +10,54 @@ const parseCookies = (cookie, key) => {
 }
 
 export const state = () => ({
-  auth: Cookie.get('auth'),
+  token: Cookie.get('token'),
+  id: Cookie.get('id'),
+  name: Cookie.get('name'),
+  email: Cookie.get('email'),
+  logo: Cookie.get('logo'),
 })
 
 export const mutations = {
-  setAuth(state, auth) {
-    Cookie.set('auth', auth)
-    state.auth = auth
+  setUserData(state, userData) {
+    if (!userData) {
+      Cookie.remove('token')
```

Comment:
I think you can create a method to delegate this responsibility: e.g. : `removeCookie`

Reply:
done